README.md

```diff
@@ -42,10 +42,10 @@ fn users_with_name(connection: &Connection, target_name: &str)
     -> Vec<(i32, String, Option<String>)>
 {
     use self::users::dsl::*;
-    users.filter(name.eq(target_name))
-        .load(connection)
-        .unwrap()
-        .collect()
+    match users.filter(name.eq(target_name)).load(connection) {
+        Ok(results) => results.collect(),
+        Err(_) => vec![]
+    }
```

Comment:
Let's just change the return type of the function to `QueryResult<Vec<(...)>>`. I think this `match` distracts from the example too much.